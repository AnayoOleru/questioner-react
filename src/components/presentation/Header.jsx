import React, { Fragment } from 'react';
import { Link } from 'react-router-dom';
import Button from './Button';

const Header = () => (
  <Fragment>
    <div className='Header'>
      <nav className='questionaire-nav'>
        <div className='nav-container'>
          <div className='nav-left'>
            <a href='./index.html'><h2>QUESTIONER</h2></a>
          </div>
          <div className='nav-right'>
            <Link to='hee'>
              <Button
                type='button'
                className='nav-button'
              >
                Logout
              </Button>
            </Link>
          </div>
        </div>
      </nav>
    </div>
  </Fragment>
);

export default Header;
