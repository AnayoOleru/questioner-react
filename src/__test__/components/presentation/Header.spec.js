import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../../components/presentation/Header';

describe('Header Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Header />);
  });
  test('Header component should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
