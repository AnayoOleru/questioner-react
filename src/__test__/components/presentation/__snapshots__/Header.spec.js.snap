// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Header component should render 1`] = `ShallowWrapper {}`;
